docs/drafts/downgrading-an-extension.md

```diff
@@ -0,0 +1,211 @@
+
+# Downgrade a ClusterExtension
+
+## Introduction
+
+Downgrading a `ClusterExtension` involves reverting the extension to a previous version. This process might be necessary due to compatibility issues, unexpected behavior in the newer version, or specific feature requirements that are only available in an earlier release. This guide provides step-by-step instructions to safely downgrade a `ClusterExtension`, including necessary overrides to bypass default constraints and disable CRD safety checks.
```

Comment:
Downgrading a `ClusterExtension` involves reverting the extension to a previous available version. This process might be required due to compatibility issues, unexpected behavior in the newer version, or specific feature requirements that are only available in an earlier release.

Reply:
Hi @rashmi43 did you mean that comment as a suggested change but didn't use the suggestion mechanism? I can't see what the difference is?

Reply:
Ah okay, they were too subtle for me before adequate coffee ☕ ☕ . Done in next commit. Thanks!